src/common/queries/expense-categories.ts

```diff
@@ -70,3 +70,15 @@ export function bulk(
     ids: id,
   });
 }
+
+export function useBlankExpenseCategoryQuery() {
+  return useQuery<ExpenseCategory>(
+    '/api/v1/expense_categories/create',
+    () =>
+      request('GET', endpoint('/api/v1/expense_categories/create')).then(
+        (response: GenericSingleResourceResponse<ExpenseCategory>) =>
+          response.data.data
+      ),
+    { staleTime: Infinity }
+  );
+}
```

Comment:
This needs an update with the latest set of permissions.

Please have a look at how we dynamically enable queries if the user has permission (src/common/queries/invoices.ts).

You'd wanna enable this query only if the user has `create_expense` permission.

Reply:
done